src/main/java/seedu/address/model/profile/ProfileItem.java

```diff
@@ -135,5 +137,10 @@ public String getItemName() {
         mapping.put("Descriptors", descriptors);
         return mapping;
     }
+
+    @Override
+    public JsonAdaptedItem getJsonAdaptedItem() {
+        return null;
```

Comment:
Is there a reason why this does not return a jsonadapted profile item?

Reply:
Ahh forgot to change it back, was testing. Same with the application item one. Thanks